screenpipe-server/src/db.rs

```diff
@@ -984,13 +988,15 @@ impl DatabaseManager {
                     r#"
                     SELECT COUNT(DISTINCT audio_transcriptions.audio_chunk_id || '_' || COALESCE(audio_transcriptions.start_time, '') || '_' || COALESCE(audio_transcriptions.end_time, ''))
                     FROM audio_transcriptions_fts
-                    JOIN audio_transcriptions ON audio_transcriptions_fts.audio_chunk_id = audio_transcriptions.audio_chunk_id
+                    JOIN audio_transcriptions ON audio_transcriptions_fts.audio_chunk_id = audio_transcriptions.audio_chunk_id          
                     WHERE {}
-                        AND (?2 IS NULL OR audio_transcriptions.timestamp >= ?2)
-                        AND (?3 IS NULL OR audio_transcriptions.timestamp <= ?3)
-                        AND (?6 IS NULL OR LENGTH(audio_transcriptions.transcription) >= ?6)
-                        AND (?7 IS NULL OR LENGTH(audio_transcriptions.transcription) <= ?7)
-                        AND (json_array_length(?8) = 0 OR audio_transcriptions.speaker_id IN (SELECT value FROM json_each(?8)))
```

Comment:
?

Reply:
let me fix that it's audio transcription and not ocr